pages/view/_iscnId.vue

```diff
@@ -51,22 +51,22 @@
               {{ $t('iscn.meta.rawData') }}
             </Button>
             <Button
+              v-if="isShowMintButton"
+              :class="['w-min', 'mr-[8px]']"
               preset="secondary"
+              :text="$t('NFTPortal.button.mint')"
+              :to="localeLocation({ name: 'nft-iscn-iscnId', params: { iscnId: iscnId } })"
+            />
+            <Button
+              preset="outline"
+              :class="['w-min', 'mr-[8px]']"
```

Comment:
Can remove the `margin-right` of the rightmost button
```suggestion
              class="w-min"
```